### ISSUE TITLE
fine-tune pretrained models with triplet loss

### ISSUE THREAD

**Issue by whu-lyh** [deepinsight/insightface#264]
hello,there. recently i'm trying to fine-tune the triplet loss for occulsion face, which pretrained model should i use, can u give me some advice? (LResNet50E-IR or LResNet34E-IR ?)

**Comment by nttstar** [comment]
any model should be ok.
